metadata-ingestion/src/datahub/ingestion/api/source_helpers.py

```diff
@@ -90,9 +91,15 @@ def auto_status_aspect(
         else:
             raise ValueError(f"Unexpected type {type(wu.metadata)}")
 
+        if wu.metadata.entityType in skip_entities:
+            # If any entity does not support aspect 'status' then skip that entity from adding status aspect.
+            # Example like dataProcessInstance doesn't suppport status aspect.
+            # If not skipped gives error: java.lang.RuntimeException: Unknown aspect status for entity dataProcessInstance
+            skip_urns.add(urn)
```

Comment:
We can say continue instead of collecting urns in skip_urns array

Reply:
At source_helpers.py:100 we are yielding all workunits which needs to ingest, hence we cannot have continue statement at source_helpers.py:98.
We can have CONTINUE logic next to source:helpers.py:102 but there we need to create logic to extract entity type from string urn. 
I think current changes are small and simple so lets keep it otherwise let me know we can modify it.

Reply:
lets discuss this in call